dev/readthedocs.md

```diff
@@ -4,28 +4,45 @@ Please visit the [docs](https://semesterly-v2.readthedocs.io/en/latest/index.htm
 answer the following questions.
 
 1. What is the command I run to get the courses from Fall 2021?
+Access the semesterly Docker container shell. Then use the command:
+`python manage.py ingest jhu --term Fall --years 2021`
+This command gets the course data from JHU, validates it, and writes it to formatted JSON
 
 2. How do I then load those courses into my database?
+`python manage.py digest jhu`
 
 3. How do I get a terminal running in my docker container?
+Click on the VSCode Docker extension, right click on the desired Docker container and click "Attach Shell".
 
 4. Where do I store data I don’t want to commit?
+You can stash data on a stack using the command `git stash`.
```

Comment:
This isn't wrong. I was referring to data such as passwords and secrets, in which git stashing it repeatedly may be a little tedious. You can find the section I want you to look at here:
https://semesterly-v2.readthedocs.io/en/latest/advancedconfig.html#overriding-setting-secrets

You can type out your revised answer in a comment to this.

Reply:
Ah I see what you were getting at, thanks. I interpreted the question as referring to git workflow.

In this case, I would store data I don't want committed in `semesterly/sensitive.py` or another ignored file.